assets/functions/packer/webpack.config.js

```diff
@@ -1,9 +1,11 @@
 module.exports = {
+  // WARNING: Do not change this section
   entry: './src/index.js',
```

Comment:
@archan937 Maybe add `target: node` to the default webpack.config.js?

https://webpack.js.org/configuration/target/

Reply:
Done, thank.